packages/isc/coreutil/coreconst.go

```diff
@@ -6,36 +6,33 @@ import (
 
 // names of core contracts
 const (
-	CoreContractDefault         = "_default"
 	CoreContractRoot            = "root"
 	CoreContractAccounts        = "accounts"
 	CoreContractBlob            = "blob"
-	CoreContractEventlog        = "eventlog"
 	CoreContractBlocklog        = "blocklog"
-	CoreContractErrors          = "errors"
 	CoreContractGovernance      = "governance"
+	CoreContractErrors          = "errors"
+	CoreContractEvm             = "evm"
```

Comment:
Nit:

```suggestion
	CoreContractEVM             = "evm"
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms